frontend/src/api/devices.js

```diff
@@ -52,6 +52,10 @@ const updateSettings = async (deviceId, settings) => {
     })
 }
 
+const startLogs = async (deviceId) => {}
+
+const stopLogs = async (deviceId) => {}
+
```

Comment:
Are these for future use Ben?

Reply:
```suggestion
```

They should be obsolete now, the counter in the websocket code should handle all that now.

Reply:
ok, which means the `exports` at lines 68,69 and the async call in https://github.com/flowforge/flowforge/blob/365bf107be8c8b46cc3c101924469fc44019fb13/frontend/src/pages/device/components/DeviceLog.vue#L52 should also be removed right?